go/libkb/nist.go

```diff
@@ -115,10 +117,19 @@ func (f *NISTFactory) NIST(ctx context.Context) (ret *NIST, err error) {
 	return f.nist, nil
 }
 
+func durationToSeconds(d time.Duration) timeUnixInSeconds {
+	return timeUnixInSeconds(d.Nanoseconds() / time.Second)
```

Comment:
no, just `d / time.Second`

Reply:
happy to change it, but i think they're the same

Reply:
i think it's clearer `d / time.Second`